### ISSUE TITLE
How about upgrade to fdk-aac 2.0.x?

### ISSUE THREAD

**Issue by nkligang** [sheinbergon/jna-aac-encoder#25]
Do you have plan to upgrade to fdk-aac 2.0.x?

**Comment by sheinbergon** [comment]
Hi

I did give it a shot 2 years ago or so.

The upgrade is not trivial, as the structure/constructs have changed, so mapping/bindings should chnage as well.
I will be giving it a look in the near future, though can't commit to exactly one.
Is using 0.1.6 not an option for you?

**Comment by nkligang** [comment]
> Hi
> 
> I did give it a shot 2 years ago or so.
> 
> The upgrade is not trivial, as the structure/constructs have changed, so mapping/bindings should chnage as well. I will be giving it a look in the near future, though can't commit to exactly one. Is using 0.1.6 not an option for you?

Hi, thank you for your reply. Maybe 0.1.6 is enough for me.

**Comment by sheinbergon** [comment]
fdk-aac 0,1.6 works well, and there already matching jars in place (assuming you're not using apple silicon). Let me know if you encounter any issue

**Comment by nkligang** [comment]
OK

**Comment by sheinbergon** [comment]
@nkligang so, good news. Your inquiry inspired me, and I think I got it working  with 2.0.2 (think).
Which OS are you testing this code on?

**Comment by nkligang** [comment]
@sheinbergon I'm very happy to hear that. I'm developer of TeamAssistor(https://www.teamassistor.com/), I found your project is because I encounter one issue: I encode audio aac using ffmpeg but failed decoding aac in Android using MediaCodec, I wonder the problem maybe is the fdk-acc algorithm, but finally I found it's not. So, fdk-aac is not need for me anymore. But the project really help me to find the solution. Thank you very much for your work!
